dev-libs/efl/efl-1.20.7.ebuild

```diff
@@ -0,0 +1,326 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="5"
+
+MY_P=${P/_/-}
+
+if [[ "${PV}" == "9999" ]] ; then
+	EGIT_SUB_PROJECT="core"
+	EGIT_URI_APPEND="${PN}"
+elif [[ *"${PV}" == *"_pre"* ]] ; then
+	MY_P=${P%%_*}
+	SRC_URI="https://download.enlightenment.org/pre-releases/${MY_P}.tar.xz"
+else
+	SRC_URI="https://download.enlightenment.org/rel/libs/${PN}/${MY_P}.tar.xz"
+	KEYWORDS="~amd64 ~arm ~hppa ~mips ~ppc ~ppc64 ~sh ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos ~x64-solaris ~x86-solaris"
+fi
+
+inherit enlightenment pax-utils gnome2-utils xdg-utils
+
+DESCRIPTION="Enlightenment Foundation Libraries all-in-one package"
+
+LICENSE="BSD-2 GPL-2 LGPL-2.1 ZLIB"
+IUSE="avahi +bmp debug connman drm +eet egl examples fbcon +fontconfig fribidi gif gles glib gnutls gstreamer harfbuzz hyphen +ico ibus jpeg2k libressl neon oldlua opengl ssl pdf physics pixman +png postscript +ppm +psd pulseaudio raw scim sdl sound svg systemd tga tiff tslib unwind v4l valgrind vlc vnc wayland webp X xcf xim xine xpm"
+
+REQUIRED_USE="
+	pulseaudio?	( sound )
+	opengl?		( || ( X sdl wayland ) )
+	gles?		( || ( X wayland ) )
+	gles?		( !sdl )
+	gles?		( egl )
+	ibus?		( glib )
+	sdl?		( opengl )
+	vnc?		( X fbcon )
+	wayland?	( egl !opengl gles )
+	xim?		( X )
+"
+
+RDEPEND="
+	avahi? ( net-dns/avahi )
+	connman? ( net-misc/connman )
+	drm? (
+		>=dev-libs/libinput-0.8
+		media-libs/mesa[gbm]
+		>=x11-libs/libxkbcommon-0.3.0
+	)
+	fontconfig? ( >=media-libs/fontconfig-2.5.0 )
+	fribidi? ( >=dev-libs/fribidi-0.19.2 )
+	gif? ( media-libs/giflib )
+	glib? ( dev-libs/glib:2 )
+	gnutls? ( >=net-libs/gnutls-3.3.6 )
```

Comment:
The ssl/gnutls logic is incorrect here. `ssl` should control using *any* SSL backend, then gnutls should toggle between GnuTLS and (Open|Libre)SSL. IOW it should be sth like:

    ssl? (
      gnutls? ( gnutls... )
      !gnutls? (
        !libressl? ( openssl... )
        libressl? ( libressl... )
       )
    )

Reply:
Done